lib/action_kit_rest/response/raise_error.rb

```diff
@@ -6,7 +6,12 @@ def on_complete(response)
         status_code = response[:status].to_i
         if (400...600).include? status_code
           if status_code == 400
-            raise ActionKitRest::Response::ValidationError.new(url: response[:url].to_s, body: response[:body])
+            response_body = response[:body]
+            if JSON.parse(response_body)['errors'] = {'mailing_id' => ['Unable to associate this mailing ID with account.']}
```

Comment:
Should this be `==` instead?

Reply:
Also, should we be checking `JSON.parse(response_body)['errors']['mailing_id']` directly instead of comparing the hashes?

Reply:
- `==`: Yes! Good catch.
- check `mailing_id` errors directly: Good question. I guess it depends whether there might be other keys, and if so how we want to handle them? My inclination is to keep this error for when the `mailing_id` error is the only one.